source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.json

```diff
@@ -159,10 +156,15 @@
     }
   },
   "productDefinitions": {
+    "filterAllProjects": "All Projects",
     "name": {
       "android_rab_s3": "Android RAB S3",
       "android_dab_s3": "Android DAB S3",
-      "android_google_play": "Android Google Play"
+      "android_google_play": "Android Google Play",
```

Comment:
I don't think it's a good thing to store readable names into the code, this should be handle by the backend.

Reply:
I disagree -- because how would the backend know what language we're using? they'd be making the same API calls we'll eventually be making -- to crowdin

unless, I'm missing something?

Reply:
Names don't need translation they are stuff name. "Android Google play" is the same in Spanish, French or other languages.